src/inputs/Select/index.tsx

```diff
@@ -64,7 +80,16 @@ function Select({ items, activeItemId, onItemClick, id, ...rest }: Props) {
             return (
               <MenuItem value={i.id} key={i.id}>
                 {i.iconUrl && <IconImg alt={i.label} src={i.iconUrl} />}
-                <span>{i.label}</span>
+                <div>
+                  {i.title && (
+                    <Text size="sm" color="secondary" strong>
+                      {i.title}
+                    </Text>
+                  )}
+                  <Text size="sm" color="text">
+                    {i.label}
+                  </Text>
```

Comment:
<img width="139" alt="Screenshot 2020-07-02 at 17 39 05" src="https://user-images.githubusercontent.com/16622558/86365744-f7601780-bc8a-11ea-929f-4e52effc485a.png">

The color seems to be the same even different props are passed, is that expected?

Reply:
do you mean the text color? Yes, it's hardcoded, do you think Title/Label colors should be customized via props?

Reply:
Title:
```<Text size="sm" color="secondary" strong>```

Label:
```<Text size="sm" color="text">```

one has color secondary and another one has color text, but the rendered colors are the same, is this expected? In general I'd expect secondary color to be more faded compared to the main color

Reply:
Also I think it makes more sense to make label a primary text (so label = title), and add a prop for secondary text (would replace label)

Reply:
that will be a "breaking visual" change. perhaps we can rename it to `mainLabel` instead of title.

Reply:
wdyt @mikheevm ?

Reply:
label should be the main text

Reply:
currently it doesn't make sense

imagine I want to display this
```
{
   "name": "ETH",
   "balance": "0.3 ETH"
}
```

so in this case, title = name, label = balance

but balance should not be used as a label, label should equal to `name` prop. And the balance is just a secondary text

Reply:
we should get rid of title and instead use label/secondary text